test/services/bundle.service.test.js

```diff
@@ -58,9 +67,19 @@ describe('Test transaction bundle upload', () => {
       });
   });
 
-  afterAll(cleanUpTest);
+  test('error thrown if method  type is not PUT or POST', async () => {
+    await supertest(server.app)
+      .post('/4_0_1/Measure/$submit-data')
+      .send(INVALID_METHOD_REQ)
+      .set('Accept', 'application/json+fhir')
+      .set('content-type', 'application/json+fhir')
+      .set('x-provenance', JSON.stringify(SINGLE_AGENT_PROVENANCE))
+      .expect(400)
+      .then(async response => {
+        expect(response.statusCode).toEqual(400);
```

Comment:
So I think we're actually hitting the wrong error case here. I believe this 400 is thrown because there is no resource attribute in the entry of the fixture used. If you add a resource attribute with a valid resourceType and id, then we'll get the expected error case, which should be verified in this test by checking the resulting error message